pages/common/restic.md

```diff
@@ -18,6 +18,10 @@
 
 `restic -r {{path/to/repository}} restore {{snapshot_id}} {{path/to/target}}`
 
+- Restore a specific file from a specific backup to a target directory:
```

Comment:
I think I should change 'specific file' to 'specific path'

Reply:
Can it restore folders too ?

Reply:
Yes. If you specify something like `/home/user` it will restore all the files matching this path. So the folder recursively.

Reply:
I just pushed a new commit with 'path' instead of 'file'.